frontend/src/lib/components/Projects/ProjectTable.svelte

```diff
@@ -43,6 +43,11 @@
         {#if isColumnVisible('users')}
           <th>{$t('project.table.users')}</th>
         {/if}
+        {#if isColumnVisible('createdAt')}
```

Comment:
I believe we sort by this value (maybe verify that). So, we should take this opportunity to add an icon symbolizing that. (Like we do on the user table and used to have on this table)

Reply:
If I see a sorting icon, I'd expect it to be clickable to reverse the sorting order. I'm inclined to *not* add the icon until we implement sorting the admin-dashboard table columns (which is out-of-scope for this Tiny-sized task, I think). But I'll go ahead and add it; we can easily remove it later on.

Reply:
Added in commit b622c47.